src/lib/components/ScrollView/ScrollView.jsx

```diff
@@ -34,9 +38,10 @@ export const ScrollView = (props) => {
     scrollbar,
     shadowColor,
     shadowSize,
-    translations,
   } = props;
 
+  const { translations } = useContext(RUIContext);
```

Comment:
Why is this written in two different ways? There is context consumer in `Modal` and direct assignment here.

Reply:
Because `ScrollView` uses the functional syntax and `Modal` uses the class syntax.